transform/format.go

```diff
@@ -121,6 +158,12 @@ func DefaultFormatValue(attr, path string, info os.FileInfo) interface{} {
 		return info.Size()
 	case "time":
 		return info.ModTime().Format(time.Stamp)
+	case "hash":
+		v, err := hash(info, path, crypto.SHA1)
+		if err != nil {
+			panic(err.Error())
```

Comment:
Not sure how I feel about panicking, but you can leave this as is for now.

Might need to refactor this method to return an error.